src/components/table/table.tsx

```diff
@@ -519,18 +627,45 @@ export class Table {
                     <limel-spinner size="large" />
                 </div>
                 {this.renderEmptyMessage()}
+                {this.renderSelectAll()}
                 <div
                     id="tabulator-table"
                     class={{
                         'has-pagination': this.totalRows > this.pageSize,
                         'has-aggregation': this.hasAggregation(this.columns),
                         'has-movable-columns': this.movableColumns,
+                        'has-rowselector': this.selectable,
+                        'has-selection': this.tableSelection?.hasSelection,
                     }}
                 />
             </div>
         );
     }
 
+    private renderSelectAll() {
```

Comment:
Yes, IIRC you were one telling me that we should introduce a specPage/table.spec.ts in a separate PR since we didn't have any tests at all for table.tsx. 
I'm all in on adding such tests (not only for the selection features) separately, but if want feel we need to do that now I can do that too.